site/src/examples/sampler/index.js

```diff
@@ -0,0 +1,399 @@
+/* global d3:false, fc:false */
+    'use strict';
```

Comment:
The indentation seems to be a bit awry, please could you reformat to use 4 spaces?